### DISCUSSION TITLE
Specific positions оf points

### DISCUSSION THREAD

**Discussion by popkovanton** [patrykandpatrick/vico#708] [Questions] [answered]
How to make the line start not from the beginning of the graph, but from a certain point. As picture
<img width="613" alt="Screenshot 2024-05-16 at 11 08 12" src="https://github.com/patrykandpatrick/vico/assets/19818752/ac769e6d-5ebf-417f-ab46-e77bd8306a23">

**Comment by Gowsky** [comment, marked as answer]
Hello! You need to modify the `minX` value. Assuming you’re using Vico  2.0.0-alpha.19, you can see how to do that [here](https://patrykandpatrick.com/vico/wiki/cartesian-charts/layers/#axisvalueoverrider).
